src/ProjectWindow.cpp

```diff
@@ -641,6 +641,7 @@ ProjectWindow::ProjectWindow(wxWindow * parent, wxWindowID id,
    mContainerWindow->Initialize(mTrackListWindow);
 
    auto effectsPanel = safenew ThemedWindowWrapper<RealtimeEffectPanel>(mContainerWindow, wxID_ANY);
+   effectsPanel->SetName(_("Realtime effects"));
```

Comment:
This long-distance coincidence of literals here and in RealtimeEffectsPanel.cpp is unfortunate.  Maybe a named constant.  Also the mismatch of capitalization would make two strings in the translation catalog.

The _ macro does a run-time translation.  If the language preference changes, is this name reset?  No.  That means the screen reader might read it in the wrong language.  Important?  Maybe not enough to fix, but comment it.

Reply:
>If the language preference changes, is this name reset? No. That means the screen reader might read it in the wrong language. Important?

As I concerned it works same way everyway, don't see why this case is special